src/handbook/marketing/hubspot.md

```diff
@@ -14,18 +14,19 @@ we talk to to ensure no crossover in sales and conversations.
 
 #### Lifecycle Stage
 
-When defining a new contact in HubSpot, an option exists to define the "Lifecycle Stage", this section
-details your options here, and when to use the respective value:
+All contacts in Hubspot need to have a defined lifecycle. It is required the all contacts pass through each lifeycle stage at the appropriate time based on their interaction with FlowFOrge marketing and sales. Contacts should not skip a stage. If is up to the contact owner to ensure their contacts have the appropriate lifecycle stage set.
```

Comment:
What about:

```suggestion
All contacts in HubSpot need to have a defined lifecycle. It is required the all contacts pass through each lifecycle stage at the appropriate time based on their interaction with FlowForge marketing and sales. If is up to the contact owner to ensure their contacts have the appropriate lifecycle stage set.
```

Reply:
I would like to keep the point contacts should not skip a stage. We need to be able to report based on when a contact enters a particular stage. If contacts skip a stage this becomes impossible to do the reporting.

Reply:
But if a contact comes in through the contact us from, they might go straight into a phase halfway the list? How do you propose to handle that?

Reply:
It is very rare that a contact comes in halfway through the lifecycle. If it does happen, what we were doing at HiveMQ is saving and changing the lifecycle stage for each step.

If we don't, it becomes difficult to answer the question 'How many MQLs did we create in [insert month]'